src/app.css

```diff
@@ -70,3 +70,8 @@ li {
 .hide-underline {
 	text-decoration: none;
 }
+
+.no-transform-button:active {
+	transform: none;
+	transition: none;
```

Comment:
Volgens mij is transition niet echt nodig. Was door die `:active` al gefixt als het goed is.